client/src/components/CFPIntro.js

```diff
@@ -6,37 +6,36 @@ const CFPIntro = () => (
   <div className="pb-6 mb-8 border-bottom">
     <h2 className="mb-4">Intro</h2>
     <h4>Welcome to Reversim Summit and thank you for submitting your proposal!</h4>
-    <p className="mb-8">Our goal is to streamline the submission and review process, while maintaining superb quality; if you have any feedback or questions, please email us at <a href={`mailto:${REVERSIM_MAIL}`}>{REVERSIM_MAIL}</a>.</p>
+    <p>
+      We encourage all people to submit a proposal to Reversim Summit 2018. If you are _considering_ submitting a proposal, we invite you to do so!
```

Comment:
Phrasing: suggest to replace `all people` with `everyone`

Reply:
done